cdk.json

```diff
@@ -16,7 +16,8 @@
       "node_modules",
       "test",
       "services/ui-src/src/**",
-      "services/ui-src/build/**"
+      "services/ui-src/build/**",
```

Comment:
FYI, the entire ui-src directory can be excluded because when we run local we run the react app outside of localstack, but so can ui-auth.  I'm going to work on this some more in a future PR.